tests/model_fields_/models.py

```diff
@@ -165,3 +165,35 @@ class Movie(models.Model):
 
     def __str__(self):
         return self.title
+
+
+class RestorationRecord(EmbeddedModel):
+    date = models.DateField()
+    restored_by = models.CharField(max_length=255)
+
+
+class ArtifactDetail(EmbeddedModel):
+    """Details about a specific artifact."""
+
```

Comment:
It's unusual for a model to have a docstrings. It looks strange, especially because the linter requires a blank line after the docstring. Instead it might be better to have a comment block above the RestorationRecord that describes how the models that follow fit together.